.github/ISSUE_TEMPLATE/Feature request form.yml

```diff
@@ -0,0 +1,82 @@
+name: Feature Request
+description: Share your idea for a new feature/API enhancement
+title: "[Feature]: "
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: 
+        Thanks for taking the time to fill out this feature request. Here is a [good sample report](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3683)
+
+  - type: textarea
+    id: feature-need
+    attributes:
+      label: Describe the feature/enhancement you need
+      description: Please describe what you want to happen, or what problem you are seeing without this feature. 
+      placeholder: I need to do xyz when ... but I'm having trouble achieving this because... 
+    validations:
+      required: true
+
+  - type: textarea
+    id: app-background
+    attributes:
+      label: About your app and what you are using WebView2 for
+      description: If possible, it would be great for us to have a better understanding of what your app do, and what you're using WebView2 in your app for. If it's been shared in another issue opened by you, you can skip this question.
+      placeholder: e.g. our app use WebView2 to handle form submission and printing in hospital kiosks.
+    validations:
+      required: false
+
+  - type: textarea
+    id: usecases
+    attributes:
+      label: The scenario/use case where you would use this feature
+      description: If possible, please shed some light on the scenario/use cases you are trying to accomplish.
+      placeholder: For example, our user need to print out a report based on a pre-defined template. We need to set the print options in a dialog box with values defined in the template. 
+    validations:
+      required: false
+
+  - type: dropdown
+    id: importance
+    attributes:
+      label: Importance
+      description: May you help us understand the importance of this ask to you?
+      options:
+        - ----Please select----
+        - Critical. My app's basic functions wouldn't work without it.
+        - Impactful. My app's user experience would be significantly compromised without it.
+        - Nice to have. There're other ways to tackle this, but having official API support would be beneficial.
+    validations:
+      required: true
+
+  - type: dropdown
+    id: timeline
+    attributes:
+      label: Timeline
+      description: Any pending timeline for your app that depend on this particular ask?
```

Comment:
I think the implications are slightly different, We'd like developers to give context about their general timeline and plan, rather than having them setting a deadline for us. Many feature requests do have a certain extent of flexibility in delivery time so I'd like to avoid using 'deadline'.